### ISSUE TITLE
Windows 11 image paths displayed incorrectly in tooltip if excessively long

### ISSUE THREAD

**Issue by IsaacHatton** [upscayl/upscayl#769]
### Checklist

- [X] I have **tried ALL** of the steps mentioned in the [troubleshooting guide](https://github.com/upscayl/upscayl/wiki/Troubleshooting).
- [X] I have **searched** for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).
- [ ] My error involves vkEnumeratePhysicalDevices or vkCreateInstance, which means I should ignore the other issues and file this one despite the last checkbox.

### Describe the Bug

Windows 11 image path with spaces displayed incorrectly

### To Reproduce

Put an image in a folder called 'new folder'
Load it into Upscayl
Hover over set output folder
Path is displayed incorrectly'

### Upscayl Version (or commit hash)

2.9.4

### Platform

Windows

### OS Version

Windows 11

### GPU Name

Intel Iris Xe

### Expected Behavior

Show full path

### Logs

```sh
Will upload if necessary for this bug report, but as this isn't a graphics or AI issue, I don't think it would help.

In the logs the path is shown correctly when the image loads into the app.
```

**Comment by aaronliu0130** [comment]
![image](https://github.com/upscayl/upscayl/assets/11874211/b613eee2-778d-46ae-9610-dcbd954f4985)
How would it be displayed incorrectly?

**Comment by IsaacHatton** [comment]
Apologies, I think I misunderstood the error, it happens with really long path lengths rather than due to the spaces.
![image](https://github.com/upscayl/upscayl/assets/74068072/31f2677f-6b46-4485-b961-7c4d82540b2f)

Correct path in Explorer breadcrumbs and as displayed in PowerShell
```powershell
C:\Users\REDACTED\Desktop\upscayl-test\qwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnm\qwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnmqwertyuiopasdfghjklzxcvbnm\qwertyuiopasdfghjklzxcvbnmq
``` 

However, Explorer's copy as path option copies the same string as what Upscayl shows.

There's a chance this is an operating system issue as well, as MS Paint struggles to "Save To" this location also.

**Comment by NayamAmarshe** [comment]
Fixed in 2.11
